server/src/tasks/deployment/email.js

```diff
@@ -0,0 +1,17 @@
+const sendgridUtils = require('@utils/sendgrid')
+
+const email = async (networks, communityProgress) => {
+  const { email, subscribe } = communityProgress.steps.email.args
+  try {
+    sendgridUtils.sendInfoMail({ email })
+    if (subscribe) {
+      sendgridUtils.subscribeUser({ email })
```

Comment:
we need to handle users that subscribe multiple times (that should not throw an error)